src/components/projects/ManuscriptPageContainer.tsx

```diff
@@ -75,10 +87,22 @@ const ManuscriptPageView: React.FC = () => {
                   <ManuscriptMenusContainerInner>
                     <ManuscriptMenus />
                   </ManuscriptMenusContainerInner>
+                  {!can.editWithoutTracking && (
+                    <>
+                      <Label>Show tracked changes</Label>
+                      <IconButton defaultColor={true} onClick={toggleTracking}>
+                        {isTrackingOn ? <SliderOnIcon /> : <SliderOffIcon />}
+                      </IconButton>
+                    </>
+                  )}
                 </ManuscriptMenusContainer>
                 {can.seeEditorToolbar && <ManuscriptToolbar />}
               </EditorHeader>
-              <EditorBody className="editor-body">
+              <EditorBody
+                className={`editor-body ${
+                  isTrackingEnabled && 'tracking-enabled'
```

Comment:
Since all the changes related to toggling this class happen inside the body editor, it seems to me, that the body-editor should toggle this class inside itself instead of article-editor controlling it from outside and potentially there can be other logic inside body editor that will depend on tracking being visible or hidden.

Reply:
I'm not sure what you are actually suggesting. Could you please elaborate?

Toggling lives in the article editor, and in the last commit, I moved the class name `tracking-visible` one level up so it's visible for all components.

Reply:
What I meant is maybe we need to toggle that className inside the body editor since the css for this class is inside the body editor

Reply:
In my opinion, having the className at a higher level is more beneficial, especially in scenarios where multiple child components need to access it, not only in the body but also in the sidebar or inspector. This approach seems simpler and reduces redundancy.

I will proceed with the current behavior as you suggested, and of course, we can change that if needed, but I don't see a need for it right now.